django_keycloak/middleware.py

```diff
@@ -42,24 +79,28 @@ def append_user_info_to_request(self, request, token):
                 user.email = user_info.get("email")
                 user.save()
 
-        except get_user_model().DoesNotExist:
-            user = get_user_model().objects.create_from_token(token)
+        except User.DoesNotExist:
+            user = User.objects.create_from_token(token)
+
+        # Add the local user to request
         request.user = user
 
         return request
 
     @staticmethod
-    def is_auth_header_missing(request):
+    def has_auth_header(request) -> bool:
         """Check if exists an authentication header in the HTTP request"""
-        return "HTTP_AUTHORIZATION" not in request.META
+        return "HTTP_AUTHORIZATION" in request.META
 
     @staticmethod
     def get_token(request):
-        """Get the token from the HTTP request"""
+        """
+        Get the token from the HTTP request
+        """
         auth_header = request.META.get("HTTP_AUTHORIZATION").split()
-        if len(auth_header) == 2:
+        if len(auth_header) > 1:
             return auth_header[1]
-        return None
+        return auth_header[0]
 
 
 class KeycloakGrapheneMiddleware(KeycloakMiddlewareMixin):
```

Comment:
Since there is a deprecation warning and a major version bump, shouldn't we remove the Graphene middleware? AFAIK it provides no additional functionality over the standard middleware but is also less efficient.

Reply:
I agree!

Reply:
@uw-rvitorino Is there anything from your side? Sorry, about all the requests for comment today :sweat_smile: